analysis/src/Commands.ml

```diff
@@ -395,7 +395,7 @@ let test ~path =
           | "dia" -> diagnosticSyntax ~path
           | "hin" ->
             let line_start = 0 in
-            let line_end = 6 in
+            let line_end = 34 in
```

Comment:
Just noticed this now -- would you add a comment about the line number hardcoded in the test?

Reply:
Done